.gitignore

```diff
@@ -3,3 +3,5 @@
 *.pyc
 
 epiquote/settings/*.py
+
+\.venv/
```

Comment:
This should go in your `.git/info/exclude`, not the gitignore, as it's something personal. Can you remove this commit? Thanks :-)